utils/config.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Config module
+ *
+ * Single point of access for the application confguration. Handles validation on required environment variables.
+ * The configuration is evaluate eagerly at the first access to the module. The module exposes convenient methods to access such value.
+ */
+
+import { tryCatch2v } from "fp-ts/lib/Either";
+import { toError } from "fp-ts/lib/Either";
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+
+// base read function
+// tslint:disable typedef
+const getEnv = () => ({
```

Comment:
an alternative to use `getRequiredStringEnv`  is to `decode(process.env)` using a type with (eventually) partials and (eventually) unions (in case of attributes dependencies). the actual code will fail on the first error, the decode can collect all errors (using Validation). aside, you don't have to type two times the name of the variables (which is error prone). what do you think about it?

Reply:
good idea